src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -173,7 +177,7 @@ int IVsEditorFactory.Close()
             return VSConstants.S_OK;
         }
 
-        private static string GetWinFormsLoaderName(IVsHierarchy vsHierarchy)
+        private string GetWinFormsLoaderName(IVsHierarchy vsHierarchy)
```

Comment:
Please annotate this as now being null returning, or rename to TryGet…

Reply:
Done!